src/components/picker/helpers/usePickerLabel.tsx

```diff
@@ -38,7 +42,9 @@ const usePickerLabel = (props: UsePickerLabelProps) => {
 
     // otherwise, extract from picker items
     const selectedItem = _.find(items, {value});
-
+    if (Constants.isWeb) {
+      return _.get(selectedItem?.value || selectedItem, 'label');
+    }
```

Comment:
Is this needed? 
We had a migration in v7 regarding the picker values and label. 
This is pure logic, I don't see a reason for web to work different from mobile in this case